common/ECChannel.cpp

```diff
@@ -68,21 +68,8 @@ HRESULT ECChannel::HrSetCtx(ECConfig *lpConfig) {
 
 	if (cert_file == nullptr || key_file == nullptr) {
 		ec_log_err("ECChannel::HrSetCtx(): no cert or key file");
-		return hr;
-	}
-	auto key_fh = fopen(key_file, "r");
-	if (key_fh == nullptr) {
-		ec_log_err("ECChannel::HrSetCtx(): cannot open key file");
```

Comment:
I think this is/was here because SSL's error messages are rather terrible. Not that this one is any better, but at least here we can add and report %s, file, strerror(errno).

Reply:
Yes, the error messages are terrible (but aren't output at the moment, anyway), but it's rather easy to get the system errno that caused SSL_CTX_SSL_CTX_use_PrivateKey_file/SSL_CTX_use_certificate_chain_file to fail in case the file couldn't be opened to have a more meaningful message output there. I'll adjust the corresponding patch in this sense.